design-library/src/components/BccItemTile/BccItemTile.stories.ts

```diff
@@ -0,0 +1,54 @@
+import { Face2Icon } from "@bcc-code/icons-vue";
+import BccItemTile from "./BccItemTile.vue";
+
+import type { Meta, StoryFn } from "@storybook/vue3";
+
+export default {
+  title: "Components/BccItemTile",
+  component: BccItemTile,
+  argTypes: {},
+} as Meta<typeof BccItemTile>;
+
+const Template: StoryFn<typeof BccItemTile> = (args) => ({
+  components: { BccItemTile },
+  setup() {
+    return { args };
+  },
+  template: `
+    <BccItemTile v-bind="args" />
+  `,
+});
+
+export const Example = Template.bind({});
+Example.args = {
+  title: "Title",
+  overline: "Overline",
+};
+
+export const WithInfoOnRight: StoryFn<typeof BccItemTile> = (args) => ({
```

Comment:
This story is wrongly named as it renders things at the bottom of the card, probably copy-paste error 😄.